libs/nacl/native_client/src/shared/imc/win/nacl_imc.cc

```diff
@@ -270,7 +270,7 @@ int NaClSendDatagram(NaClHandle handle, const NaClMessageHeader* message,
     return -1;
   }
   if (0 < message->handle_count && message->handles) {
-    HANDLE target;
+    HANDLE target = NULL;
```

Comment:
Why using C's convention of NULL instead of C++'s standard nullptr?
This should not block, though, and if there is no issue, I would happily change NULL to nullptr myself. This occurrence and the others.

Reply:
Mostly to be consistent with source file style of using `NULL`/`nullptr` :)

Will change to a modern one, thought, thanks!

Reply:
Well, you do the hard work here, so don't feel obligated to repatch, it's a matter of a sed regex after all.

Reply:
Ok, have started to rebase, but feel too lazy. Will stay as is, may be owners replace in future.